webpack.config.js

```diff
@@ -0,0 +1,117 @@
+"use strict";
+
+const CleanWebpackPlugin  = require('clean-webpack-plugin'),
+  HtmlWebPackPlugin       = require('html-webpack-plugin'),
+  MiniCssExtractPlugin    = require('mini-css-extract-plugin'),
+  OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin'),
+  Package                 = require('./package.json'),
+  path                    = require('path'),
+  project_config          = require('./config.js'),
+  UglifyJsPlugin          = require('uglifyjs-webpack-plugin');
+
+const versionJS        = Package.webpack_bundle_version_js,
```

Comment:
It's unnecessary for this to be a new `const`.  Just remove the semi colon from the one above and continue.  I also prefer either of these styles for variable delcaration:

```js
const CleanWebpackPlugin = require('clean-webpack-plugin')
    , HtmlWebPackPlugin = require('html-webpack-plugin')
    , MiniCssExtractPlugin = require('mini-css-extract-plugin')
```
OR

```js
const CleanWebpackPlugin = require('clean-webpack-plugin');
const HtmlWebPackPlugin = require('html-webpack-plugin');
const MiniCssExtractPlugin = require('mini-css-extract-plugin');
```